app/views/visits/_form.html.erb

```diff
@@ -1,10 +1,14 @@
 <%= bootstrap_form_with(model: [@district, @facility, @patient, visit], local: true, label_errors: true) do |form| %>
   <div class="row">
     <div class="col-md-6">
-      <%= form.date_field :measured_on, label: "Date attended", required: true, autofocus: true,
-                          help: "Format: DD/MM/YYYY", min: '2018-01-01', max: '2019-01-01', value: visit.measured_on || '2018-01-01' %>
-      <%= form.number_field :systolic, required: true, min: 50, max: 400 %>
-      <%= form.number_field :diastolic, required: true, min: 30, max: 400 %>
+      <% if visit.new_record? && @previous_visit.present? %>
+        <p class="bg-light p-3 text-muted">Previous date attended: <%= @previous_visit.measured_on %></p>
+      <% end %>
+
+      <%= form.text_field :measured_on, label: "Date attended", required: true, autofocus: true, placeholder: "DD/MM/YYYY",
+                          help: "Format: DD/MM/YYYY", value: visit.measured_on || @previous_visit&.measured_on || @patient.registered_on %>
```

Comment:
- Do we prefer `text_field` instead of `date_field`? Why?
- We could probably extract `default_measured_on` to the controller to capture domain logic more cohesively.
- Is `@previous_visit&` a typo?

Reply:
`text_field` is better, as we can force it to use our DD/MM/YYYY format. `date_field` doesn't allow you to specify data formats, as the browser uses the system format.

There's definitely a better way to handle defaults, but this is fine for now.

`&.` is a ruby operator that acts like `try()`

Reply:
Oh, `date_field` puts a date picker on the interface, and lets the user tab between date, month, year, even without entering slashes, which is quite useful. Does text field with a pattern also give that?

Reply:
Just trust me :)

I spoke with Deepa and she nor the data entry people are using the datepicker, and the confusion around date formats was a good reason to change it.